libmscore/system.cpp

```diff
@@ -1090,7 +1090,9 @@ void System::scanElements(void* data, void (*func)(void*, Element*), bool all)
                         }
                   v = v1 || v2; // hide spanner if both chords are hidden
                   }
-            if (all || (score()->staff(staffIdx)->show() && _staves[staffIdx]->show() && v) || spanner->isVolta())
+            if (all || 
+                (score()->staff(staffIdx)->show() && _staves[staffIdx]->show() && v && !spanner->isTie()) || 
```

Comment:
According to the [MuseScore coding rules](https://musescore.org/en/handbook/developers-handbook/finding-your-way-around/musescore-coding-rules#Coding-style), multi-line conditionals should be indented three additional spaces, not four, from the second line on.

Reply:
Which is half the 'usual' indent

Reply:
I think it's best to start the three-indent lines with `||`'s instead of putting them at the end of the lines.

Reply:
That's not what they told me 9 months ago.  I prefer it that way too and I was told to put the logical operators at the end of each line.

Reply:
OK, suit yourself. There're no documented rules on that.

Reply:
I'm just trying to avoid resubmitting.  Last time I submitted a PR with logical operators at the beginning of lines in a multi-line if statement, I was told to change them and put them on the previous line at the end.  I'm just following orders.

Reply:
The rules didn't used to be so complete and detailed.  It's possible that this multi-line thing just hasn't made it into the written rules.  Maybe one of the rule-writers can chime in on that.